GliaWidgetsTests/CoreSDKClient.Failing.swift

```diff
@@ -74,6 +74,10 @@ extension CoreSdkClient {
         },
         unsubscribeFromUpdates: { _, _ in
             fail("\(Self.self).unsubscribeFromUpdates")
+        },
+        subscribeForUnreadSCMessageCount: { _ in
+            fail("\(Self.self).subscribeForUnreadSCMessageCount")
+            return UUID().uuidString
```

Comment:
Regular empty string would also fit the purpose:
```suggestion
            return ""
```

But if `UUID` is really needed it is better to use mock:

```suggestion
            return UUID.mock.uuidString
```

Reply:
Fixed https://github.com/salemove/ios-sdk-widgets/pull/1112/commits/f99771f3e8f5efd277ab6d66978888c311137bea